templates/networkpolicies.yaml

```diff
@@ -90,6 +90,19 @@ spec:
       ports:
         - protocol: TCP
           port: 5432
-  # Deny all egress traffic
-  egress: []
+  # Allow egress to istio service mesh. This shouldn't cause any issues if
+  # Istio is not in use.
+  egress:
+  - to:
+    - namespaceSelector:
+        matchLabels:
+          app.kubernetes.io/name: istio-controlplane
+    - podSelector:
+        matchLabels:
+          istio: pilot
+  - ports:
```

Comment:
this should also be part of the "to", not a separate item - this allows outbound port 53 to anything (check the describe output)

Reply:
This is a really handy tool to visualize how the policies work - unfortunately the policy language is kind of badly designed, which leads to a lot of gotchas :frowning: https://editor.cilium.io/

Reply:
more info about this behavior here: https://kubernetes.io/docs/concepts/services-networking/network-policies/#behavior-of-to-and-from-selectors

Reply:
I tested this out and the two podSelectors can be combined, but we need to allow all outbound TCP/UDP traffic for DNS to function.